### PR TITLE
Created automated linter workflow

### PR DESCRIPTION
**Pull request by maaverik**
Modeled after https://github.com/samuelmeuli/lint-action

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Lint
+
+on: push
```
Comment (daltonfury42) [inline .github/workflows/lint.yml:3]: let's do on all PRs to master:
```
on:
  pull_request:
    branches: [master]
```

Then we can add a branch protection rule to allow merging only if the check passes

**Comment by daltonfury42** [comment]
It's breaking though

**Comment by maaverik** [comment]
From what I understand, it breaks cause it expects eslint and prettier to be present already, but that's not the case for this commit.

**Comment by maaverik** [comment]
It still doesn't detect eslint and prettier. I thought it uses the same package.json as the project, but still fails saying eslint is not found. Have to do more research.

**Pull request merged**